tox.ini

```diff
@@ -35,3 +33,6 @@ minversion=2.0
 addopts= -rxsX 
 pep8ignore = E501 E128 E127
```

Comment:
those errors disappeared with the changes, i removed the setting since its no longer needed